v2/internal/test/errstrings/err_posix.go

```diff
@@ -0,0 +1,9 @@
+//go:build !windows
+// +build !windows
+
+// OS specific error strings
+package errstrings
```

Comment:
This isn't my favorite thing, but I think is fine in the short term. I think we should probably change the tests to uses `errors.Is` or something to detect the right error condition, but I recognize that is a larger change.